### DISCUSSION TITLE
auth issue

### DISCUSSION THREAD

**Discussion by Nuub073** [hlf20010508/telegram-onedrive#19] [Q&A] [answered]
- server_uri=https://xxxx:8000
      # - reverse_proxy=$reverse_proxy(optional, true or false, default to false)
      - tg_bot_token=xxxxxiuA
      - tg_api_id=10xxxx
      - tg_api_hash=404xxxxc9d
      - tg_user_phone=+xxxxx
      # - tg_user_password=$tg_user_password(optional, if you have two-step verification enabled)
      # - tg_user_name=$tg_user_name(optional, no @)
      - od_client_id=3exxxxxx
      - od_client_secret=xxxxxx
      - remote_root_path=Documents/downloads
      # - delete_flag=$delete_flag(optional, true or false, default to false)
    command: python main.py

I am deploying this on oracle cloud free tier, the port is open there,  i have  checked redirect uri is with https and /auth at the end in azure, however when i do the /auth command n then open my server url , it never opens, not in chromium or any other browser. Every variable is correctly entered, id secret api hash etc

**Comment by hlf20010508** [comment]
- Did you open up your port `8000` on your oracle machine?
- Was your port reflection in configuration correct like this?
  ```yml
  ports:
    - 8000:8080
  ```
- By the way, your `remote_root_path` should be `/Documents/downloads`.

**Comment by Nuub073** [comment, marked as answer]
> * Did you open up your port `8000` on your oracle machine?
> * Was your port reflection in configuration correct like this?
>   ```yaml
>   ports:
>     - 8000:8080
>   ```
> * By the way, your `remote_root_path` should be `/Documents/downloads`.

hii , thanks for pointing out my mistake, i had changed the ports to 8000:8000 because 8080 wasnot free, freed it n its running now, and remote path too, i noticed it later n fixed it, many thanks sir for this repo, i had been looking for a bot that can upload tg files to onedrive ever since heroku went paid n mirror bots died, thank you so much
